README.md

```diff
@@ -1,5 +1,109 @@
-# audit-log-plugin
+# Audit Logging Plugin
 
+Project Status
+---
 [![Build Status](https://travis-ci.org/jvz/audit-log-plugin.svg?branch=master)](https://travis-ci.org/jvz/audit-log-plugin)
 
-Audit logging plugin for Jenkins based on Apache Log4j Audit
+#### License [license] | Cover Coverage Status (https://todothis.com) | Maven Central Location (https://todothis.com)
+
+## Project Overview
+
+In [Jenkins][jenkins-site], there are several categories of user-initiated actions, such as:
+- Starting and/or stopping a Jenkins build.
+- Performing a change to the Jenkins configuration settings.
+- TODO: Add some other user-actions
+
+As well as actions not initiated by the user, such as by the Jenkins system, for example:
```

Comment:
This is a sentence fragment. It would be better to convert this to a proper sentence (subject - verb).